src/components/dev-hub/survey-banner.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import Button from './button';
+import Icon from '@leafygreen-ui/icon';
+import { fontSize, screenSize, size, colorMap } from './theme';
+
+const Footer = styled('div')`
+    display: ${({ showSurvey }) => (showSurvey ? `block` : `none`)};
+    background-color: ${colorMap.greyDarkTwo};
```

Comment:
We should use the `theme` in this component if possible (even though this is quick/dirty it may still persist when we re-skin the DevHub in Q3)

Reply:
done